code/modules/reagents/chemistry/machinery/chem_master.dm

```diff
@@ -17,12 +17,12 @@
 	var/useramount = 30 // Last used amount
 	layer = BELOW_OBJ_LAYER
 
-/obj/machinery/chem_master/New()
+/obj/machinery/chem_master/Initialize()
 	create_reagents(100)
 	add_overlay("waitlight")
 	var/obj/item/weapon/circuitboard/machine/B = new /obj/item/weapon/circuitboard/machine/chem_master(null)
 	B.apply_default_parts(src)
-	..()
```

Comment:
This was running code before parent New() that would now happen after New().

Does this matter in this case?